File: Services/Services.Application/Models/ServiceModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Services.Application.Models
{
    public record ServiceModel(Guid Id, string ServiceName, Guid ServiceCategoryId, Guid SpecializationId, decimal Price, bool IsActive) : BaseModel(Id);
}

File: react-client/src/models/services/ServiceModel.ts
import ServiceCategoryModel from "@models/serviceCategories/serviceCategoryModel";
import SpecializationModel from "@models/specializations/specializationModel";

export default interface ServiceModel {
  id: string,
  serviceName: string,
  serviceCategory: ServiceCategoryModel,
  specialization: SpecializationModel,
  price: number,
  isActive: boolean
}